common/src/main/java/com/wynntils/mc/mixin/ItemStackMixin.java

```diff
@@ -60,6 +61,14 @@ private TooltipFlag onGetTooltipLines(TooltipFlag flags) {
         return event.getFlags();
     }
 
+    @ModifyReturnValue(method = "copy", at = @At("RETURN"))
```

Comment:
Can you use that semi-qualified mixin format we use? I think I've seen you use it in your other mixins, but it's basically the fully qualified method target without the L<classpath>; prefix.

Reply:
Those are because we have been sloppy in reviewing, combined with that we have no automatic checks. The format bolyai described is the one we should be using.